Integrations/integration-RTIR.yml

```diff
@@ -426,138 +498,165 @@ script:
         else:
             demisto.results('No attachments found.')
 
+
+    def get_ticket_history_by_id(ticket_id, history_id):
+        """Accepts ticket ID and history ID as input and returns a dictionary of ticket history entry properties"""
+
+        suffix_url = 'ticket/{}/history/id/{}'.format(ticket_id, history_id)
+        raw_history = http_request('GET', suffix_url, params={'user': USERNAME, 'pass': PASSWORD})
+
+        return raw_history
+
+
     def get_ticket_history(ticket_id):
-        raw_history = tracker.get_history(ticket_id=ticket_id)
-        output_history = []
-        headers = ['Created', 'Creator', 'Description']
-        for history in raw_history:
-            current_history = {
-                'Created': history['Created'],
-                'Creator': history['Creator'],
-                'Description': history['Description']
-            }
-            content = history.get('Content')
-            if content != 'This transaction appears to have no content':
-                current_history['Content'] = content
-                if not 'Content' in headers:
-                    headers.append('Content')
-            new_value = history.get('NewValue')
-            if new_value:
-                current_history['NewValue'] = new_value
-                if not 'NewValue' in headers:
-                    headers.append('NewValue')
-            output_history.append(current_history)
-        return output_history, raw_history, headers
+
+        suffix_url = 'ticket/{}/history'.format(ticket_id)
+        raw_history = http_request('GET', suffix_url, params={'user': USERNAME, 'pass': PASSWORD})
+        history_context = []
+        headers = ['id','Created', 'Creator', 'Description']
```

Comment:
initialize `headers` before entry
this should be `id`, add white space after comma